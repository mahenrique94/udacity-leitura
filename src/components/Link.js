import styled from 'styled-components'

import { Link } from 'react-router-dom'

const StyledLink = styled(Link)`
    color: #FDFDFD;
    text-decoration: none;
    &:hover {
        color: #FDFDFD;
    }
`

export default StyledLink
