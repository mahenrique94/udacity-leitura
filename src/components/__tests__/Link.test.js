import React from 'react'
import renderer from 'react-test-renderer'

import Link from '../Link'

describe('Tests for Link component', () => {

    it('Should rendering without crash', () => {
        const tree = renderer.create(
            <Link>Test</Link>
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })

})
