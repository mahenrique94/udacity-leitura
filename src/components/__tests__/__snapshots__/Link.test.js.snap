// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Link component Should rendering without crash 1`] = `
<a
  href="/"
>
  Test
</a>
`;
